internal/terraform/node_resource_abstract_instance.go

```diff
@@ -865,6 +865,9 @@ func (n *NodeAbstractResourceInstance) plan(
 		providers.ValidateResourceConfigRequest{
 			TypeName: n.Addr.Resource.Resource.Type,
 			Config:   unmarkedConfigVal,
+			ClientCapabilities: providers.ClientCapabilities{
+				WriteOnlyAttributesAllowed: true,
```

Comment:
Same question here: Do we not need to be passing the other capability here as well? i.e. `DeferralAllowed`?